src/app_service/service/settings/service.nim

```diff
@@ -484,6 +484,16 @@ QtObject:
       return true
     return false
 
+  proc isSepoliaEnabled*(self: Service): bool =
+    return self.settings.isSepoliaEnabled
+
+  proc toggleIsSepoliaEnabled*(self: Service): bool =
+    let newValue = not self.settings.isSepoliaEnabled
+    if(self.saveSetting(KEY_IS_SEPOLIA_ENABLED, newValue)):
+      self.settings.isSepoliaEnabled = newValue
```

Comment:
Add this line:
`self.events.emit(SIGNAL_USE_OF_SEPOLIA_NETWORK_UPDATED, SettingsBoolValueArgs(value: self.settings. isSepoliaEnabled))`